tests/Request/VisionRequestTest.php

```diff
@@ -12,22 +16,28 @@ class VisionRequestTest extends \PHPUnit_Framework_TestCase
      */
     protected $image;
 
+    /**
```

Comment:
You can remove these empty blocks. I personally only add them when there are returns or parameters.